components/tools/OmeroWeb/test/integration/test_tags.py

```diff
@@ -167,7 +167,7 @@ def test_add_edit_and_remove_tag(self):
         _post_response(self.django_client, request_url, data)
         _csrf_post_response(self.django_client, request_url, data)
         # Check that tag is removed - short delay to allow async delete
-        sleep(0.1)
+        sleep(1)
```

Comment:
Slight heads up that we can only handle so many calls to `sleep(second or more)`. Eventually, this would be better handled with a `@retry` (of some form) on the assertion itself.